pkgs/applications/kde/marble.nix

```diff
@@ -9,9 +11,10 @@ let
       name = "marble";
       meta.license = with lib.licenses; [ lgpl21 gpl3 ];
 
-      nativeBuildInputs = [ extra-cmake-modules ];
+      nativeBuildInputs = [ extra-cmake-modules kdoctools perl ];
       propagatedBuildInputs = [
-        qtscript qtsvg qtquickcontrols
+        qtdoc qtscript qtsvg qtquickcontrols qtwebkit shared_mime_info
```

Comment:
I think `qtdoc` is usually only needed during the build and not at runtime. If that's true here, could you move this to `nativeBuildInputs`?